tests/ballerina-test/src/test/java/org/ballerinalang/test/nativeimpl/functions/UtilTest.java

```diff
@@ -116,7 +117,62 @@ public void testHMACValueFromBase64ToBase16Encoding() {
         BValue[] args = new BValue[]{new BString("Hello Ballerina"), new BString("abcdefghijk")};
         BValue[] returnValues = BRunUtil.invoke(compileResult, "testHMACValueFromBase64ToBase16Encoding", args);
         Assert.assertFalse(returnValues == null || returnValues.length == 0 || returnValues[0] == null,
-                           "Invalid return value");
+                "Invalid return value");
         Assert.assertEquals(returnValues[0].stringValue(), expectedValue);
     }
+
```

Comment:
Include test cases for negative and invalid numbers as well. i.e: `hexToDecimal("xyz")` , `decimalToHex(-12345)`

Reply:
Done!